azure-cpi.html.md.erb

```diff
@@ -115,7 +117,7 @@ Example of 10GB disk:
 ```yaml
 disk_pools:
 - name: default
-  disk_size: 10_240
+  disk_size: 10_240 # On Azure the disk size must be in the range [1 * 1024, 1023 * 1024].
```

Comment:
How about as below?

Example of 10GB disk:

* **disk_size** [Integer, required]: Size of the disk in megabytes. On Azure the disk size must be in the range [1 * 1024, 1023 * 1024].

```yaml
disk_pools:
- name: default
  disk_size: 10_240
```